src/lib.rs

```diff
@@ -76,8 +76,13 @@
 
 extern crate futures as lib_futures;
 extern crate rand;
+
+#[cfg(feature = "spin_mutex")]
 extern crate spin;
 
+#[cfg(feature = "parking_lot_mutex")]
+extern crate parking_lot as spin;
+
```

Comment:
Would be more accurate with the following IMHO:

```suggestion
#[cfg(feature = "spin_mutex")]
extern crate spin as sync;

#[cfg(feature = "parking_lot_mutex")]
extern crate parking_lot as sync;
```